devices.js

```diff
@@ -145,7 +145,7 @@ const devices = [
         model: 'WXKG03LM',
         vendor: 'Xiaomi',
         description: 'Aqara single key wireless wall switch',
-        supports: 'single click',
+        supports: 'single, double, hold, release and long click',
```

Comment:
I don't believe this is correct. I just received my WXKG03LM today and I certainly can't get it to report anything besides a single click no matter how fast I click.
`zigbee2mqtt:debug 3/25/2019, 11:31:25 PM Received zigbee message of type 'attReport' with data '{"cid":"genOnOff","data":{"onOff":0}}' of device 'lumi.sensor_86sw1lu' (0x00158d000231a2c8)`

Long click results in the above message as soon as the button is pressed down and seems to initiate an unpairing process if held down for more than 5 seconds.
Looking [here](https://github.com/dresden-elektronik/deconz-rest-plugin/issues/811) it seems there are old/new versions of the switch? 
I suggest we do something to not confuse people like me 😄

Reply:
Yeah true, it's just documented for the dual switch. You can create a PR if you want.

http://www.zigbee2mqtt.io/information/supported_devices.html#xiaomi

The new version has an higher delay to support the double click. They are unusable for my case. I think this is also a relevant info.

Reply:
https://github.com/Koenkk/zigbee-shepherd-converters/blob/cb1d5f0f96d81710bc04052f61985a0dc182a340/devices.js#L186

@Zagitta Here you can find the two different IDs for the new and the old device model.